hack/generator/pkg/codegen/resource_registration_file.go

```diff
@@ -142,7 +151,24 @@ func makeClientGoKubernetesSchemePackageReference() astmodel.PackageReference {
 	return astmodel.MakeExternalPackageReference("k8s.io/client-go/kubernetes/scheme")
 }
 
-// createGetKnownTypesFunc creates a getKnownTypes function:
+// createGetKnownStorageTypesFunc creates a getKnownStorageTypes function that returns all storage types:
+//		func getKnownStorageTypes() []runtime.Object {
+//			var result []runtime.Object
+//			result = append(result, new(<package>.<resource>))
+//			result = append(result, new(<package>.<resource>))
+//			result = append(result, new(<package>.<resource>))
+//			...
+//			return result
+//		}
+func (r *ResourceRegistrationFile) createGetKnownStorageTypesFunc(codeGenerationContext *astmodel.CodeGenerationContext) (dst.Decl, error) {
+	return createKnownTypesFuncImpl(
+		codeGenerationContext,
+		r.storageVersionResources,
```

Comment:
Where do we ensure these are always in the same order, so that codegen is deterministic?

Reply:
Inside the function itself, line 206